### ISSUE TITLE
CODEONERS file only shows @Mahad as owner 

### ISSUE THREAD

**Issue by gbowne1** [taotnpwaft/website#12]
The CODEOWNERS file only appears to show @Mahad410  as owner with shield as owner of this project, even though it lists the others, but they do not have shields next to their names.

Ideas @hitblast ?

Side note still need to work on building an active website team.

**Comment by hitblast** [comment]
Could you show me a visual representation for the issue?

To be a part of the CODEOWNERS file, one must have write access to that particular repository. From my end, it shows that the file has errors.

**Comment by gbowne1** [comment]
Yeah, this should show up in reviews mode during a commit, PR, Merge cycle.  

It shows errors on my side too. 

If theres a CODEOWNERS file it will display in one of the frames / panels on the side of the page as required for review, etc.

I need to look at the docs for this more.

**Comment by hitblast** [comment]
> Yeah, this should show up in reviews mode during a commit, PR, Merge cycle.
> 
> It shows errors on my side too.
> 
> If theres a CODEOWNERS file it will display in one of the frames / panels on the side of the page as required for review, etc.
> 
> I need to look at the docs for this more.

Alright, I'm converting some of the issues here to discussions for a detailed overview of them.
